release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -59,6 +59,11 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 [id="ocp-4-12-web-console"]
 === Web console
 
+[id="ocp-4-12-developer-perspective"]
+==== Developer Perspective
+
+* With this update, you can export your application in the ZIP file format to another project or cluster using the *+Add* view or the *Topology* view. 
```

Comment:
may be ?
```suggestion
* With this update, you can export your application in the ZIP file format to another project or cluster using the newly added `Export application` card on the *+Add* page or the `Export Application` button on the *Topology* view. 
```
And the `Export Application` option was already there on the topology page, we simply added a card to the Add page

Reply:
@debsmita1, I have tweaked the line a bit. Also, if the Export application buuton is not newly added in the topology view with this release, we can skip that part as we have to highlight only the new features.